pyproject.toml

```
@@ -1,2 +1,2 @@
 [build-system]
-requires = ["numpy", "setuptools", "wheel"]
+requires = ["oldest-supported-numpy", "setuptools", "wheel"]
```

I suggest also adding the build-backend information while we're here:

```suggestion
requires = ["oldest-supported-numpy", "setuptools", "wheel"]
build-backend = 'setuptools.build_meta'
```